src/graphql/resolvers/queries/event.js

```diff
@@ -51,18 +51,18 @@ export const fieldResolvers = {
           })),
         );
     },
-    sessions: (
+    sessions: async (
       { id },
       { onOrAfter, daysAfter },
       { dataSources: { firestore } },
     ) => {
       dlog('sessions');
 
-      return sessionStore(firestore).findAllApprovedByEventIdAtDate(
-        id,
-        onOrAfter,
-        daysAfter,
-      );
+      const result = await sessionStore(
```

Comment:
total nit.. but if we want to keep things similar then just chain a .then like we did above and drop the async/await. end result is the same but jsut consistent.

Reply:
I agree with this nit

Reply:
fixed in commit 05d7e4d